### ISSUE TITLE
Use github for hosting the website

### ISSUE THREAD

**Issue by scravy** [ellson/graphviz#88]
I guess that would save you a lot of trouble. graphviz.github.io sounds not too bad.

**Comment by ErwinJanssen** [comment]
I was thinking about this as well, but the Graphviz site also includes a forum, whereas Github Pages are only static sites right? Still, could be helpful for hosting most of the documentation (which is also included with the software I believe), which can then also be improved and edited by the community. What do the maintainers think? Something we can help with?

**Comment by magneticnorth** [comment]
Yes, we have a bunch of content in drupal forums, and other content in mailman,
and bug reports in Mantis.

We would like to

**Comment by emdenrg** [comment]
Until we get a new or better home, it might make sense to use github as 
a secondary site, for the documentation and binary packages. The 
documentation is already html. We would just lose the drupal envelope.

**Comment by GadgetSteve** [comment]
@ErwinJanssen  It could be well worth looking at the github [help ](https://help.github.com/articles/using-jekyll-as-a-static-site-generator-with-github-pages/) on the use of Jekyll for static site generation and/or take a look at the python [Hyde](https://github.com/hyde/hyde). 

Either can make regenerating the web pages and publishing them to github, (or a later site), easy.

**Comment by ErwinJanssen** [comment]
The pull request I created earlier is a quick and dirty way to get the documentation on Github pages.

@GadgetSteve Those links look promising, maybe it can be used to improve the documentation when it is decided Github pages will be a more permanent solution.

Hosting the documentation on a Github pages might make it easier to maintain and more people can contribute as well.

**Comment by ellson** [comment]
Erwin,

Thanks very mich for this ... committed.

        http://ellson.github.io/graphviz/

We've just made a decision to migrate graphviz.org onto GitHub (subject 
to any feedback from GitHub admins).

This is a great starting point.

John

**Comment by GadgetSteve** [comment]
John @ellson & @ErwinJanssen  - Switching the website to Jekyll/Hyde could be well worth the effort whichever hosting solution is decided on as:

 - the editable "Source" becomes a set of reStructuredText rather than html which is easier to edit
 - in Hyde the Diagrams can actually be _re_-generated from dot files automatically so would automatically reflect the current processing of the examples _this makes it really easy to spot when an exaple is broken or produces unexpected results_
 - re-buiding and publishing becomes a single command whichever hosting solution you use
 - only the changed files are normally ge-generated & uploaded _think make for a website_
 - changes in style are quick and easy

**Comment by emden** [comment]
Actually, most of the documentation web pages are generated from simpler 
data, and the figures are also generated. See doc/infosrc.

**Comment by GadgetSteve** [comment]
A total of 1032 tickets transferred.

**Comment by clemens-tolboom** [comment]
Checking ie #1131 guess the import didn't worked well
![_webdot__webdot_in_perl_doesn_t_handle_generalized_url_syntax_ _issue__1131_ _ellson_graphviz](https://cloud.githubusercontent.com/assets/371014/16577619/88cb7b74-4296-11e6-8cd6-13d043a1c1b1.png)

It contains a link to `https://github.com/~north/cgi-bin/webdot1.7.11/webdot.cgi//~north/webdot1.7.11/tut2.dot.dot.map`

**Comment by GadgetSteve** [comment]
An automated script can only do so much - I cannot get to the original xml from work at the moment as guess what - the original is on graphvis.com - I will try to get a chance to take a look at this specific one tonight **but** I can only go in what was in the xml export which does not include:

 - Images
 - Attachments
 - Font changes
 - etc.
 
**Please**, if anybody notices errors or omissions from specific ported tickets point them out and I will attempt to address them manually - I personally do not have the time to go through 1032 tickets looking for issues.

Note that some issues **will** result from example code being interpreted as markdown.

**Comment by clemens-tolboom** [comment]
Hmmm ... can't you archive / webcrawler the old issues into a static HTML site and put it in a new repo on github instead? Adding 'bad' issues to the issue queue will make issue triage a pitb. Or is the old server/site completely dead?

**Comment by GadgetSteve** [comment]
@clemens-tolboom - the old issues site is not reliably accessible _or more accurately is reliably inaccessible_ - this is the whole point of the move to github.  What I got was an xml export from that server. While I haven't had the time to look at 100% of the transposed issues those that I have looked at seem to have been cleanly migrated _other than this one_ - While I agree that this is a PITB it is probably better for most people than a bug tracker that only people who do not have any available time have access to.

**Comment by GadgetSteve** [comment]
I have just completed a run though the ported bugs looking for older issues that match git log entries with "Fix bug" followed by an Original ID that was in the ported Issues list and have marked those that matched closed. This moved about 80 to closed.
